src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--5 .stars__star,
+.stars--4 .stars__star:not(:last-child),
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n+3),
+.stars--3 .stars__star:nth-child(-n+4) {
```

Comment:
Fix this logic 
![image](https://github.com/mate-academy/layout_catalog/assets/70009205/8a18405f-e965-41ed-8fa3-8a8b85dcd084)